src/videojs-resumecard.js

```diff
@@ -73,146 +72,39 @@ class ResumeCard {
-        return restartButton;
-      }
-    }
-
-    restartButton = document.createElement('button');
-
-    restartButton.className = [
-      classNameButton,
-      classNameRestartButton,
-    ].join(' ');
-    restartButton.innerText = restartButtonText;
-    restartButton.onclick = callback;
-    restartButton.type = 'button';
-
-    return restartButton;
-  }
-
-  /**
-   * Generates a resume button.
-   *
-   * @param {resumeCallback} callback
-   * @returns {HTMLElement} resumeButton
-   */
-  getResumeButton(callback) {
-    const {
-      classNameButton,
-      classNameResumeButton,
-      getResumeButton,
-      resumeButtonText,
-    } = this.settings;
-
-    let resumeButton;
-
-    if (typeof getResumeButton === 'function') {
-      resumeButton = getResumeButton.call(this, callback);
-
-      if (resumeButton instanceof HTMLElement) {
-        return resumeButton;
-      }
+  hide() {
+    if (resumeCard instanceof HTMLElement === false) {
+      return;
     }
 
-    resumeButton = document.createElement('button');
-
-    resumeButton.className = [
-      classNameButton,
-      classNameResumeButton,
-    ].join(' ');
-    resumeButton.innerText = resumeButtonText;
-    resumeButton.onclick = callback;
-    resumeButton.type = 'button';
-
-    return resumeButton;
+    resumeCard.style.display = 'none';
   }
 
-  /**
-   * Generates the resume card template.
-   *
-   * @param {HTMLElement} restartButton
-   * @param {HTMLElement} resumeButton
-   * @returns {HTMLElement} template
-   */
-  getTemplate(restartButton, resumeButton) {
+  render() {
     const {
       className,
-      classNameActionItem,
-      classNameActionList,
-      getTemplate,
       id,
+      template,
     } = this.settings;
 
-    let template;
+    resumeCard = document.createElement('div');
+    resumeCard.className = className;
+    resumeCard.id = id;
 
-    if (typeof getTemplate === 'function') {
-      template = getTemplate.apply(this, [restartButton, resumeButton]);
-
-      if (template instanceof HTMLElement) {
-        return template;
-      }
+    if (typeof template !== 'function') {
+      this.player.el().appendChild(resumeCard);
     }
 
-    template = document.createElement('div');
-
-    template.className = className;
-    template.id = id;
-
-    const actionList = document.createElement('ul');
-
-    actionList.className = classNameActionList;
-
-    const actionItemRestart = document.createElement('li');
-
-    actionItemRestart.className = classNameActionItem;
-    actionItemRestart.appendChild(restartButton);
-
-    const actionItemResume = document.createElement('li');
-
-    actionItemResume.className = classNameActionItem;
-    actionItemResume.appendChild(resumeButton);
-
-    actionList.appendChild(actionItemRestart);
-    actionList.appendChild(actionItemResume);
-
-    template.appendChild(actionList);
-
-    return template;
-  }
+    const resumeCardTemplate = template(
+      this.restartCallback.bind(this),
```

Comment:
wouldn't it be better to do the `.bind(this)` in the `constructor` so that we only have to do it once, instead of every call of `render`.  Not sure if we ever call `render` more than once, but it also feels like a more standard place to do that.